src/main/java/net/schmizz/sshj/DefaultConfig.java

```diff
@@ -84,6 +85,30 @@ public DefaultConfig() {
         setKeepAliveProvider(KeepAliveProvider.HEARTBEAT);
     }
 
+    /**
+     * Default SLF4J-based implementation of the SSHJ LoggerFactory.
+     */
+    public static class DefaultLoggerFactory implements LoggerFactory {
```

Comment:
Oops, no, I forgot to remove that.

> On Aug 24, 2016, at 6:46 AM, Jeroen van Erp <notifications@github.com> wrote:
> 
> In src/main/java/net/schmizz/sshj/DefaultConfig.java <https://github.com/hierynomus/sshj/pull/267#discussion_r76041428>:
> 
> > @@ -84,6 +85,30 @@ public DefaultConfig() {
> >          setKeepAliveProvider(KeepAliveProvider.HEARTBEAT);
> >      }
> >  
> > +    /**
> > +     * Default SLF4J-based implementation of the SSHJ LoggerFactory.
> > +     */
> > +    public static class DefaultLoggerFactory implements LoggerFactory {
> Is this one still needed?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/hierynomus/sshj/pull/267/files/f63a88ec9f1ca21db2d18eb68056fcf51a107ff5#r76041428>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABIPfZA7VswXT9_xQW3HZvhzKb2HmCGqks5qjC8DgaJpZM4JrhJ->.
>